07 - Arrays/Program.cs

```diff
@@ -5,7 +5,179 @@ namespace _07___Arrays
 {
     class Program
     {
-        
+        static void Main(string[] args)
+        {
+            // Тут многомерный массив с полями
+            string[,] ticTacToe = new string[3, 3]
+            {
+                {"1", "2", "3"},
+                {"4", "5", "6"},
+                {"7", "8", "9"}
+            };
+
+            // это отрисовка его в консоли
+            void ShowDeck()
+            {
+                Console.WriteLine("     |     |     ");
+                Console.WriteLine("  {0}  |  {1}  |  {2}  ", ticTacToe[0, 0], ticTacToe[0, 1], ticTacToe[0, 2]);
+                Console.WriteLine("_____|_____|_____");
+                Console.WriteLine("     |     |    ");
+                Console.WriteLine("  {0}  |  {1}  |  {2}  ", ticTacToe[1, 0], ticTacToe[1, 1], ticTacToe[1, 2]);
+                Console.WriteLine("_____|_____|_____");
+                Console.WriteLine("     |     |     ");
+                Console.WriteLine("  {0}  |  {1}  |  {2}  ", ticTacToe[2, 0], ticTacToe[2, 1], ticTacToe[2, 2]);
+                Console.WriteLine("     |     |     ");
+            }
+
+
+            bool isSquareTaken = false;
+            //тут я храню поле, которое выбрал юзер
+            int chosenSquare;
+            // тут я храню, сколько ходов уже сделанно
+            int turnNumber = 0;
+            //тут отмечены номера очередей, когда когда должен ходить X
+            int[] turnForX = { 1, 3, 5, 7, 9 };
+
+
+            void CheckForWin()
+            {
+                string winMessage = "Player '{0}' is the winner!";
+
+                // Diagonal from the left
+                if (ticTacToe[0, 0] == ticTacToe[1, 1] && ticTacToe[1, 1] == ticTacToe[2, 2])
```

Comment:
please implement make it's easy:
1) for diag is ok
2) for lines make 'for' with counter
3) for coulumns also make 'for'